server/src/copilot/api/openai.ts

```diff
@@ -0,0 +1,156 @@
+import axios from 'axios';
```

Comment:
Do we really need axios, now that fetch is built into both node and the browser?

Reply:
I have removed Axios and replaced it with the built-in fetch API for HTTP requests as this reduces dependency overhead and also there is no significant advantage of using axios.